### DISCUSSION TITLE
api연동

### DISCUSSION THREAD

**Discussion by jujwahun** [commerce-api-naver/commerce-solutions#71] [묻고 답하기]
어플리케이션 등록시 제 스토어선택창이 뜨지않습니다.
통합매니저 확인까지 했는데 어떻게 처리해야될지 문의드립니다.

**Comment by commerce-api-naver** [comment]
안녕하세요. 커머스API 이민석입니다.

이 공간은 커머스솔루션 연동 기술지원 공간입니다.
말씀하신 내용은 스마트스토어 판매자 분들께서 자신의 스토어에 API를 직접 연동하는 목적으로 사용하는 '내 스토어 애플리케이션'에 대한 문의로 추정됩니다.

문의주신 내용이 솔루션 연동과 관련된 기술문의일 경우 좀 더 세부적인 내용을 알려주시기 바랍니다.
그렇지 않고 커머스API센터에서 '내 스토어 애플리케이션' 발급 과정에서 어려움을 겪고 계신다면 [스마트스토어 고객센터](https://help.sell.smartstore.naver.com)로 커머스API센터 이용과 관련한 문의를 진행해보시기 바랍니다.
